src/style.css

```diff
@@ -2,0 +1,85 @@
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy, Arial, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar {
+  margin: 0;
+}
+
+.input {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+
+  font-family: Avenir, Avenir-Heavy, Arial, Helvetica, sans-serif;
+  color: #3d4e61;
+}
+
+.search-bar__big {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding: 25px 0 26px 62px;
+  background: url('./images/Search.svg') no-repeat 26px center;
+
+  background-size: 19px;
+  margin-block: 20px;
```

Comment:
`margin-block` is not a standard CSS property. Did you mean to use `margin` or `margin-block-start` and `margin-block-end`? (CSS issue)